src/renderer/components/atoms/text/helper/buildStylesFor.ts

```diff
@@ -0,0 +1,14 @@
+import type { TextUsage, TextVariant } from '../types';
+import { DOCUMENT_VARIANTS } from './constants';
+import buildStylesForDocument from './buildStylesForDocument';
+import buildStylesForInterface from './buildStylesForInterface';
+
+const buildStylesFor = (variantName: TextVariant, as: TextUsage): string => {
+  if (DOCUMENT_VARIANTS.includes(as)) {
+    return buildStylesForDocument(as);
+  }
+
+  return buildStylesForInterface(variantName, as);
+};
```

Comment:
This is probably a nitpick, but I think that to keep things consistent, you should rewrite this to something like:
```suggestion
const buildStylesFor = (variantName: TextVariant, as: TextUsage): string => {
  if (DOCUMENT_VARIANTS.includes(as)) {
    return buildStylesForDocument(as);
  } else if (INTERFACE_VARIANTS.includes(as)) {
    return buildStylesForInterface(variantName, as);
  } else {
    throw new Error("'as' prop does not have a valid value")
  }
};
```

That way the function would yell at us and we can spot any error in all this logic

Reply:
done!